lib/push.js

```diff
@@ -177,6 +181,39 @@ module.exports = function (log, db) {
     },
 
     /**
+     * Notifies every device in the account except the current one that the
+     * password was changed
```

Comment:
This is because the action we do on the browser when we get a reset pwd push message is to reset the cache credentials and transition the browser UI in "please reconnect" mode, which we don't want since we already got the new credentials with webchannel